src/main/resources/test_files/runtime-spark/RumbleML/EstimatorTests/MLEstimator-BucketedRandomProjectionLSH.jq.disabled

```diff
@@ -0,0 +1,22 @@
+(:JIQS: ShouldRun; Output="({ "label" : 0, "name" : "a", "age" : 20, "weight" : 50, "selectedFeatures" : [ 50 ] }, { "label" : 1, "name" : "b", "age" : 21, "weight" : 55.3, "selectedFeatures" : [ 55.3 ] }, { "label" : 2, "name" : "c", "age" : 22, "weight" : 60.6, "selectedFeatures" : [ 60.6 ] }, { "label" : 3, "name" : "d", "age" : 23, "weight" : 65.9, "selectedFeatures" : [ 65.9 ] }, { "label" : 4, "name" : "e", "age" : 24, "weight" : 70.3, "selectedFeatures" : [ 70.3 ] }, { "label" : 5, "name" : "f", "age" : 25, "weight" : 75.6, "selectedFeatures" : [ 75.6 ] })" :)
+let $data := annotate(
+    json-file("./src/main/resources/queries/rumbleML/sample-ml-data-flat.json"),
+    { "label": "integer", "binaryLabel": "integer", "name": "string", "age": "double", "weight": "double", "booleanCol": "boolean", "nullCol": "null", "stringCol": "string", "stringArrayCol": ["string"], "intArrayCol": ["integer"],  "doubleArrayCol": ["double"],  "doubleArrayArrayCol": [["double"]] }
+)
+
+let $est := get-estimator("BucketedRandomProjectionLSH")
+let $tra := $est(
+    $data,
+    { "inputCol": "REQUIRES A VECTOR HERE" }
```

Comment:
Very good -- knowing this will help with our design and this is where (hidden) sparse arrays will be useful.